_summaries/wu2021rethinking_2.md

```diff
@@ -1,19 +1,90 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: wu2021rethinking
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: aabayomi
+score: 10
 ---
 
 TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+
+### What is the core idea?
+
+The authors argues using positive random orthogonal features can estimate softmax showing speed and performance gains over SOTA transformer architecture
+
+### How is it realized (technically)?
+
+[1] Only non-negative kernel scores are useful.
+
+<img width="700px" src="wu2021rethinking_2h.png"/>
+
+ <!-- x,y \
+``` math
+SM (x,y) = Eω∼N(0,Id) exp ω x− 2 exp ω y− 2 = ΛEω∼N(0,Id) cosh(ω z), (
+
+where Λ = exp(−∥x∥2+∥y∥2 ) and cosh is hyperbolic cosine. Consequently, softmax-kernel admits a
+ 2
+positive random feature map unbiased approximation with h(x) = exp(− ∥x∥2 ), l = 1, f1 = exp 2
+and D = N(0,Id) or: h(x) = √1 exp(−∥x∥2 ), l = 2, f1(u) = exp(u), f2(u) = exp(−u) and the 22
++
+􏱂m same D (the latter for further variance reduction). We call related estimators: SM
+hyp+
+􏱂m
+``` -->
+
+[2] Use of orthogonal random features (ORFs)
+
+Well known method to reduce the variance of the estimator  by selecting random features and orthogonal random samples. 
```

Comment:
It's kind of difficult to follow this line, maybe you can give some context.  Also, add about the representational power of FAVOR+ on large scale tasks.